services/case-ms/src/lambdas/updatePdf.ts

```diff
@@ -1,13 +1,25 @@
+/* eslint-disable no-console */
 import to from 'await-to-js';
 
-import config from '../../../config';
+import config from '../libs/config';
```

Comment:
Not sure if this is a good way to salve it using symlinks?

Please let's have a talk :)

Reply:
typescript expects source code to be located in a single folder tree, usually with the root named src. Crosslinking outside this path comes with a lot of caveats and hence symlinking is preferred.